### PR TITLE
WIP google-differential-privacy

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@jakirkham

**Comment by jakirkham** [comment]
cc @conda-forge/staged-recipes

File: recipes/google-differential-privacy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: bazel build differential_privacy/...
+
+requirements:
+  build:
+    - bazel
+    - {{ compiler('cxx') }}
+    - openjdk
+    - pip
+    - python
```
Comment (jakirkham) [inline recipes/google-differential-privacy/meta.yaml]: Why are `pip` and `python` in `build`?

File: recipes/google-differential-privacy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - postgresql
```
Comment (jakirkham) [inline recipes/google-differential-privacy/meta.yaml]: Why is `postgresql` in `build`?

Comment (mwilson8) [reply recipes/google-differential-privacy/meta.yaml]: just based off of the languages listed on the homepage, I was assuming they'd be needed

**Comment by mwilson8** [comment]
@jakirkham removing those lines causes a error about not finding `postgres`

**Comment by jakirkham** [comment]
Unfortunately a bit saturated ATM.

Let's see if anyone else has ideas. @conda-forge/staged-recipes?

**Comment by mwilson8** [comment]
re-ping @conda-forge/staged-recipes any help is greatly appreciated

**Comment by mwilson8** [comment]
@ocefpaf @jjhelmus the [tensorflow-hub-feedstock](https://github.com/conda-forge/tensorflow-hub-feedstock/blob/0c7acb881aacadbb1bb0fe5d151c49f8351f66c1/recipe/build.sh#L5) is the only other feedstock I found that uses `bazel` to build; I'm having trouble with this one. Would you be able to provide any assistance?

**Comment by mariusvniekerk** [comment]
You probably need to set `CMAKE_CXX_FLAGS` and some others

**Comment by mwilson8** [comment]
> You probably need to set `CMAKE_CXX_FLAGS` and some others

@mariusvniekerk I've never had a recipe where I need to do that.. could you explain the purpose/usage?

**Comment by mwilson8** [comment]
as of the most recent commit, I'm not receiving an error about not being able to find the "readline" package, although I have it included in the meta file.. any suggestions?

**Comment by mwilson8** [comment]
From what I can tell, this error is caused from `postgres`.. presumably while it's trying to build from source..? @mariusvniekerk as a maintainer on postgres did you ever encounter this error?

**Comment by mwilson8** [comment]
It seems that the [conda-forge version of bazel](https://github.com/conda-forge/bazel-feedstock) is way behind the [current version](https://github.com/bazelbuild/bazel/releases).. this could be the root cause, but unsure

**Pull request closed without merging**